### PR TITLE
Added logs for redshift

### PR DESCRIPTION
**Pull request by achettyiitr**
**Fixes** # (*issue*)

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: warehouse/redshift/redshift.go [hunk 1]
```diff
@@ -435,6 +435,7 @@ func (rs *HandleT) loadUserTables() (errorMap map[string]error) {
 		return
 	}
 
+	pkgLogger.Infof(fmt.Sprintf("RS: Starting create table for users with sqlStatement: %s", sqlStatement))
```
Comment (lvrach) [inline warehouse/redshift/redshift.go:438]:
```suggestion
	pkgLogger.Infof("RS: Starting create table for users with sqlStatement: %s", sqlStatement)
```

File: warehouse/redshift/redshift.go [hunk 2]
```diff
@@ -443,11 +444,13 @@ func (rs *HandleT) loadUserTables() (errorMap map[string]error) {
 		errorMap[warehouseutils.UsersTable] = err
 		return
 	}
+	pkgLogger.Infof("RS: Competed create table for users")
 	defer rs.dropStagingTables([]string{stagingTableName})
 
 	primaryKey := "id"
 	sqlStatement = fmt.Sprintf(`DELETE FROM %[1]s."%[2]s" using %[1]s."%[3]s" _source where (_source.%[4]s = %[1]s.%[2]s.%[4]s)`, rs.Namespace, warehouseutils.UsersTable, stagingTableName, primaryKey)
 
+	pkgLogger.Infof(fmt.Sprintf("RS: Starting dupup table for users using delete sqlStatement: %s", sqlStatement))
```
Comment (lvrach) [inline warehouse/redshift/redshift.go:453]:
```suggestion
	pkgLogger.Infof("RS: Starting dupup table for users using delete sqlStatement: %s", sqlStatement)
```

File: warehouse/redshift/redshift.go [hunk 3]
```diff
@@ -475,6 +480,7 @@ func (rs *HandleT) loadUserTables() (errorMap map[string]error) {
 		errorMap[warehouseutils.UsersTable] = err
 		return
 	}
+	pkgLogger.Infof("RS: Completed records for table:%s\n", warehouseutils.UsersTable, sqlStatement)
```
Comment (lvrach) [inline warehouse/redshift/redshift.go:483]:
```suggestion
	pkgLogger.Infof("RS: Completed records for table:%s, query: %s\n", warehouseutils.UsersTable, sqlStatement)
```

File: warehouse/redshift/redshift.go (hunk 1 above)
Comment (achettyiitr) [reply warehouse/redshift/redshift.go]: Done.

File: warehouse/redshift/redshift.go (hunk 2 above)
Comment (achettyiitr) [reply warehouse/redshift/redshift.go]: Done.

File: warehouse/redshift/redshift.go (hunk 3 above)
Comment (achettyiitr) [reply warehouse/redshift/redshift.go]: Done.

**Comment by achettyiitr** [comment]
Closing this as we have found the issue what hyperverge is having.

**Pull request closed without merging**